src/i18n.js

```diff
@@ -339,6 +339,33 @@ const resources = {
       select: 'Dil Seçin',
     },
   },
+  'hi-IN': {
+    translation: {
+      title: 'वाईफाई कार्ड',
+      'desc.use':
+        'अपने वाईफाई लॉगिन की जानकारी एक साधारण कार्ड पे प्रिंट करे। अपने फ्रिज पर लगाएं, अपने बटुए में रखें, आदि।',
+      'desc.privacy':
+        'आपके वाईफाई की जानकारी कभी किसी सर्वर पर नहीं भेजी जाती। इस वेबसाइट पर ट्रैकिंग , एनालिटिक्स या फिंगरप्रिंटिंग का इस्तेमाल नहीं होता।',
+      'desc.source': 'सोर्स कोड देखो',
+      'wifi.login': 'वाईफाई लॉगिन',
+      'wifi.name': 'नेटवर्क का नाम',
+      'wifi.name.placeholder': 'वाईफाई नेटवर्क का नाम',
+      'wifi.password': 'पासवर्ड',
+      'wifi.password.placeholder': 'पासवर्ड',
+      'wifi.password.hide': 'प्रिंट करने से पहले पासवर्ड छुपाएं',
+      'wifi.password.encryption': 'एन्क्रिप्शन',
+      'wifi.tip':
+        'अपने आप कनेक्ट होने के लिए अपने फ़ोन के कैमरे से QR कोड को स्कैन करें',
+      'wifi.alert.name': 'नेटवर्क का नाम खाली नहीं हो सकता',
+      'wifi.alert.password.length.5':
+        'पासवर्ड कम से कम 5 अक्षरों का होना चाहिए',
```

Comment:
```suggestion
        'पासवर्ड कम से कम ५ अक्षरों का होना चाहिए',
```

I suggest using native hindu numbers as well! ( :, you're the expert tho

Reply:
I won't suggest this change.
I have not seen this numerals being used a lot.
All the major Hindi newspaper use `1234567890` , even google when used in Hindi uses the same . 
Same with the my phone when i switch language to Hindi.